build.gradle

```diff
@@ -610,6 +610,19 @@ project('spring-integration-mongodb') {
 	}
 }
 
+project('spring-integration-r2dbc') {
+	description = 'Spring Integration R2DBC Support'
+	dependencies {
+		api project(':spring-integration-core')
+		api ('org.springframework.data:spring-data-r2dbc') {
+			exclude group: 'org.springframework'
+		}
+		testCompile group: 'io.r2dbc', name: 'r2dbc-h2', version: '0.8.4.RELEASE'
+		testCompile group: 'com.h2database', name: 'h2', version: '1.4.200'
```

Comment:
Please, use `testImplementation` instead.
I also wonder why do we need this dependency since `r2dbc-h2` should bring it for us transitively.
Isn't it? See Maven repo for that one: https://mvnrepository.com/artifact/io.r2dbc/r2dbc-h2/0.8.4.RELEASE

Reply:
Updated build.gradle to use variable r2dbch2version and also removed r2dbc-h2 dependency. You are right io.r2dbc does bring com.h2database as transitive dependency
<img width="1680" alt="Transitive dependecy" src="https://user-images.githubusercontent.com/18318051/85644038-7fc33480-b65b-11ea-95ec-c5010ff13296.png">